packages/near-snap/src/lib/near-snap.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  WalletModuleFactory,
+  WalletBehaviourFactory,
+  InjectedWallet,
+} from "@near-wallet-selector/core";
+import { isMobile } from "is-mobile";
+import { enable, getSnapAccounts, signTransactions } from "./methods";
+import { isMetaMaskAvailable } from "./utils";
+
+const isDev = false;
+
+const NearSnapWallet: WalletBehaviourFactory<InjectedWallet> = async ({
+  options,
+  provider,
+  store,
+}) => {
+  const network = options.network.networkId as "testnet" | "mainnet";
+
+  return {
+    async signIn() {
+      // Sign in to My Wallet for access to account(s).
+      await enable(isDev);
+
+      return await getSnapAccounts(isDev, network);
+    },
+
+    async signOut() {
+      // Nothing to do here!!
```

Comment:
Why is this function empty?
How do we sign out from an account in Near Snap, and later on signIn into another account?

Reply:
Can we have a bit of info/insight on how `signOut` works here?

Reply:
> Why is this function empty? 

There is no dedicated signout method that can be called programmatically, and as we do not store anything in the state there is no need to clean. Only `signIn` is important to request authorization for the page.

> How do we sign out from an account in Near Snap, and later on signIn into another account?

Currently, snaps do not have implemented a multi-account feature like a regular MetaMask but hope it will be possible to do in future

Reply:
Thank you, for the explanation.